### ISSUE TITLE
The loss is still about 2 when it's about 4000 iterations.

### ISSUE THREAD

**Issue by myhooo** [zhengyang-wang/Deeplab-v2--ResNet-101--Tensorflow#32]
Hello, thank you for your kind to public your implementation, it's a great work and very helpful for me. When I train the model on the cityscapes dataset, the loss almost can't decrease after 100 iterations and it's about 2. The mIoU can increase slowly(about 80 iterations per 0.001 increasing). I want to know if it's normal? If it's abnormal, do you have advice to me about what things might cause this? 
Thanks in advance.^_^
![image](https://user-images.githubusercontent.com/18443212/35785011-1792828c-0a58-11e8-83cb-054a576e0bb7.png)
![image](https://user-images.githubusercontent.com/18443212/35785052-5d25bea4-0a58-11e8-9592-5490db8e0be8.png)

**Comment by zhengyang-wang** [comment]
Obviously, some modifications were made on my code. Since then I cannot answer your question.
Why does the training process involve the computation of mIoU?
In my case, the original setting for PASCAL works well for Cityscapse. Increasing the size of training patches and changing the learning rate accordingly can result in improved performance.

**Comment by myhooo** [comment]
Ok. Thank you for your reply. I think I should learn your code more carefully. ^_^

**Comment by zhengyang-wang** [comment]
You are welcome to ask questions if any part of my code appears hard to understand. Your questions may help others. Thanks.
